sdk/search/azure-search/src/main/java/com/azure/search/models/ValueFacetResult.java

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search.models;
+
+
+/**
+ * Contains value facet found by a search query
+ */
+public class ValueFacetResult {
```

Comment:
This should be annotated with `@Immutable` given that this class only has getters and all properties are `final`.